### DISCUSSION TITLE
How to make JSON files Into Bruno Collections?

### DISCUSSION THREAD

**Discussion by tinker123456** [usebruno/bruno#3117] [Q&A]
My company has a large set of collections and environments that it maintains in Git.

I haven't been able to import them directly into Bruno.

I've had to first export them from Postman and I would like to cut Postman out of the picture.

What would I need to do to make those JSON files into Bruno collections and Bruno environments?

**Comment by mbaudis** [comment]
Upvote here. I just want to generate .bru files from whatever objects exist in a Python stack or as generic YAML/JSON files. YAML and JSON I/O are a given - but there are no generic

* YAML[^1] => .bru
* JSON => .bru
* Python => .bru

... options AFAIK. [fastbruno](https://github.com/fastbruno/fastbruno) doesn't count since it is not generic (probably one can deparse to get ideas but ...). Easy to write a use case specific converter but really not the way to go...


[^1]: Still don't get why not using YAML which has excellent multiline support etc.

**Reply by helloanoop** [reply]
@mbaudis Supporting `yaml` as a storage format instead of `.bru` is on our [roadmap](https://www.usebruno.com/roadmap). Hoping to have it release within the next 2 months.

**Comment by helloanoop** [comment]
For folks bumping into this thread, I recommend checking out our `@usebruno/converters` SDK to programatically convert postman/insomnia collections into bruno - https://docs.usebruno.com/converters/overview
